packages/admin-ui/src/radio/radio-group.tsx

```diff
@@ -0,0 +1,80 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+import { createComponent, useElement } from '@vtex/admin-ui-react'
+import type { RadioState } from 'ariakit/radio'
+import { RadioGroup as AriakitRadioGroup } from 'ariakit/radio'
+
+import { Text } from '../components/Text'
+import { Set } from '../components/Set'
+import { Message } from './message'
+
+export const RadioGroup = createComponent<'fieldset', RadioGroupOptions>(
+  (props) => {
+    const {
+      label,
+      helpText,
+      errorText,
+      children,
+      direction = 'row',
+      state,
+      error = false,
+      ...restProps
+    } = props
+
+    const orientation = direction === 'row' ? 'horizontal' : 'vertical'
+
+    return useElement('fieldset', {
+      ...restProps,
+      children: (
+        <Set spacing="$l" orientation="vertical">
```

Comment:
Just worried with the `Set` component, because Matheus is changing to Stack and probably this would require updates here.

Reply:
done @Lenderson